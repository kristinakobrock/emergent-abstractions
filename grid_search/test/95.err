  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:01, 24.19it/s] 14%|#############                                                                                | 7/50 [00:00<00:01, 26.64it/s] 20%|##################4                                                                         | 10/50 [00:00<00:01, 26.19it/s] 26%|#######################9                                                                    | 13/50 [00:00<00:01, 27.29it/s] 32%|#############################4                                                              | 16/50 [00:00<00:01, 26.28it/s] 38%|##################################9                                                         | 19/50 [00:00<00:01, 24.71it/s] 44%|########################################4                                                   | 22/50 [00:00<00:01, 26.04it/s] 50%|##############################################                                              | 25/50 [00:01<00:01, 23.45it/s] 56%|###################################################5                                        | 28/50 [00:01<00:00, 23.97it/s] 64%|##########################################################8                                 | 32/50 [00:01<00:00, 26.82it/s] 70%|################################################################3                           | 35/50 [00:01<00:00, 24.39it/s] 76%|#####################################################################9                      | 38/50 [00:01<00:00, 25.44it/s] 82%|###########################################################################4                | 41/50 [00:01<00:00, 25.55it/s] 88%|################################################################################9           | 44/50 [00:01<00:00, 25.51it/s] 94%|######################################################################################4     | 47/50 [00:01<00:00, 25.45it/s]100%|############################################################################################| 50/50 [00:02<00:00, 23.32it/s]100%|############################################################################################| 50/50 [00:02<00:00, 24.96it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 29.70it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 29.87it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 25.88it/s]100%|############################################################################################| 13/13 [00:00<00:00, 27.58it/s]100%|############################################################################################| 13/13 [00:00<00:00, 27.69it/s]
