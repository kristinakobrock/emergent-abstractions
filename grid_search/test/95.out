# ["--attributes=5", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=128", "--learning_rate=0.001", "--hidden_size=256", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.8018178939819336, "acc": 0.5107726454734802, "length": 3.3360543251037598, "mode": "train", "epoch": 1}
{"loss": 0.6631532311439514, "acc": 0.6292968392372131, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6926738023757935, "acc": 0.6231383681297302, "length": 1.8581291437149048, "mode": "train", "epoch": 2}
{"loss": 0.6741846799850464, "acc": 0.6064453721046448, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6702685356140137, "acc": 0.5990159511566162, "length": 1.080627202987671, "mode": "train", "epoch": 3}
{"loss": 0.6640887260437012, "acc": 0.62890625, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6656150817871094, "acc": 0.622913122177124, "length": 1.0376380681991577, "mode": "train", "epoch": 4}
{"loss": 0.6638948321342468, "acc": 0.6181640625, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6615333557128906, "acc": 0.6195117235183716, "length": 1.02621591091156, "mode": "train", "epoch": 5}
{"loss": 0.6609218716621399, "acc": 0.6275390982627869, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6601487398147583, "acc": 0.6228487491607666, "length": 1.0197246074676514, "mode": "train", "epoch": 6}
{"loss": 0.6632782816886902, "acc": 0.616015613079071, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6596506237983704, "acc": 0.6211198568344116, "length": 1.016767978668213, "mode": "train", "epoch": 7}
{"loss": 0.6609106659889221, "acc": 0.6267578601837158, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.6589439511299133, "acc": 0.622823178768158, "length": 1.0149911642074585, "mode": "train", "epoch": 8}
{"loss": 0.6615381240844727, "acc": 0.6214843392372131, "length": 1.0, "mode": "test", "epoch": 8}
{"loss": 0.6574756503105164, "acc": 0.6247987151145935, "length": 1.0124568939208984, "mode": "train", "epoch": 9}
{"loss": 0.6604832410812378, "acc": 0.624804675579071, "length": 1.0, "mode": "test", "epoch": 9}
{"loss": 0.6577399969100952, "acc": 0.6247339248657227, "length": 1.0117629766464233, "mode": "train", "epoch": 10}
{"loss": 0.6603639125823975, "acc": 0.6220703125, "length": 1.0, "mode": "test", "epoch": 10}
{"loss": 0.6576167941093445, "acc": 0.6244306564331055, "length": 1.0102218389511108, "mode": "train", "epoch": 11}
{"loss": 0.6611590385437012, "acc": 0.6216797232627869, "length": 1.0, "mode": "test", "epoch": 11}
{"loss": 0.6570975184440613, "acc": 0.6247953772544861, "length": 1.009236454963684, "mode": "train", "epoch": 12}
{"loss": 0.6600989103317261, "acc": 0.6228516101837158, "length": 1.0, "mode": "test", "epoch": 12}
{"loss": 0.6573742628097534, "acc": 0.6239186525344849, "length": 1.0088680982589722, "mode": "train", "epoch": 13}
{"loss": 0.6608428955078125, "acc": 0.6201171875, "length": 1.0, "mode": "test", "epoch": 13}
{"loss": 0.6572446823120117, "acc": 0.6239209175109863, "length": 1.0084924697875977, "mode": "train", "epoch": 14}
{"loss": 0.6612926125526428, "acc": 0.619140625, "length": 1.0, "mode": "test", "epoch": 14}
{"loss": 0.6571683883666992, "acc": 0.6237906813621521, "length": 1.0069587230682373, "mode": "train", "epoch": 15}
{"loss": 0.6610105633735657, "acc": 0.6224609017372131, "length": 1.0, "mode": "test", "epoch": 15}
{"loss": 0.6566486954689026, "acc": 0.6250359416007996, "length": 1.00650155544281, "mode": "train", "epoch": 16}
{"loss": 0.6596897840499878, "acc": 0.6242188811302185, "length": 1.0, "mode": "test", "epoch": 16}
{"loss": 0.6566982269287109, "acc": 0.6253960132598877, "length": 1.0056663751602173, "mode": "train", "epoch": 17}
{"loss": 0.659334659576416, "acc": 0.623242199420929, "length": 1.0, "mode": "test", "epoch": 17}
{"loss": 0.6569761037826538, "acc": 0.6249266266822815, "length": 1.0054771900177002, "mode": "train", "epoch": 18}
{"loss": 0.66139817237854, "acc": 0.6224609017372131, "length": 1.0, "mode": "test", "epoch": 18}
{"loss": 0.6563279032707214, "acc": 0.6244245767593384, "length": 1.004955530166626, "mode": "train", "epoch": 19}
{"loss": 0.6604340076446533, "acc": 0.6236328482627869, "length": 1.0, "mode": "test", "epoch": 19}
{"loss": 0.6563789248466492, "acc": 0.6247868537902832, "length": 1.004216194152832, "mode": "train", "epoch": 20}
{"loss": 0.6598423719406128, "acc": 0.623046875, "length": 1.0, "mode": "test", "epoch": 20}
{"loss": 0.6566460728645325, "acc": 0.6254068613052368, "length": 1.0040735006332397, "mode": "train", "epoch": 21}
{"loss": 0.6618475317955017, "acc": 0.6234374642372131, "length": 1.0, "mode": "test", "epoch": 21}
{"loss": 0.6561912298202515, "acc": 0.6248989105224609, "length": 1.0038484334945679, "mode": "train", "epoch": 22}
{"loss": 0.6597955822944641, "acc": 0.6251954436302185, "length": 1.0, "mode": "test", "epoch": 22}
{"loss": 0.6557353138923645, "acc": 0.6248334050178528, "length": 1.0033841133117676, "mode": "train", "epoch": 23}
{"loss": 0.6594028472900391, "acc": 0.625, "length": 1.0, "mode": "test", "epoch": 23}
{"loss": 0.6560096144676208, "acc": 0.6244471073150635, "length": 1.0031886100769043, "mode": "train", "epoch": 24}
{"loss": 0.6613130569458008, "acc": 0.6216797232627869, "length": 1.0, "mode": "test", "epoch": 24}
{"loss": 0.6561613082885742, "acc": 0.6247906684875488, "length": 1.0028541088104248, "mode": "train", "epoch": 25}
{"loss": 0.6611390113830566, "acc": 0.621289074420929, "length": 1.0, "mode": "test", "epoch": 25}
{"loss": 0.656340479850769, "acc": 0.6260434985160828, "length": 1.0026228427886963, "mode": "train", "epoch": 26}
{"loss": 0.660213828086853, "acc": 0.6253905892372131, "length": 1.0, "mode": "test", "epoch": 26}
{"loss": 0.6560269594192505, "acc": 0.6245526671409607, "length": 1.0023702383041382, "mode": "train", "epoch": 27}
{"loss": 0.6606345772743225, "acc": 0.6255859732627869, "length": 1.0, "mode": "test", "epoch": 27}
{"loss": 0.6559016108512878, "acc": 0.6249651908874512, "length": 1.0022838115692139, "mode": "train", "epoch": 28}
{"loss": 0.6616177558898926, "acc": 0.619335949420929, "length": 1.0, "mode": "test", "epoch": 28}
{"loss": 0.6556238532066345, "acc": 0.6251897811889648, "length": 1.002020001411438, "mode": "train", "epoch": 29}
{"loss": 0.6608028411865234, "acc": 0.6224609017372131, "length": 1.0, "mode": "test", "epoch": 29}
{"loss": 0.6549372673034668, "acc": 0.6254792809486389, "length": 1.0020097494125366, "mode": "train", "epoch": 30}
{"loss": 0.6599246263504028, "acc": 0.625195324420929, "length": 1.0, "mode": "test", "epoch": 30}
{"loss": 0.6553573608398438, "acc": 0.6250849962234497, "length": 1.001875638961792, "mode": "train", "epoch": 31}
{"loss": 0.6608240604400635, "acc": 0.6232421398162842, "length": 1.0, "mode": "test", "epoch": 31}
{"loss": 0.6548089385032654, "acc": 0.6257727146148682, "length": 1.001652479171753, "mode": "train", "epoch": 32}
{"loss": 0.6597790718078613, "acc": 0.624804675579071, "length": 1.0, "mode": "test", "epoch": 32}
{"loss": 0.6550248861312866, "acc": 0.6256008148193359, "length": 1.001479148864746, "mode": "train", "epoch": 33}
{"loss": 0.6606409549713135, "acc": 0.6228516101837158, "length": 1.0, "mode": "test", "epoch": 33}
{"loss": 0.6553367376327515, "acc": 0.6232451796531677, "length": 1.0013275146484375, "mode": "train", "epoch": 34}
{"loss": 0.6634668111801147, "acc": 0.6140623688697815, "length": 1.0, "mode": "test", "epoch": 34}
{"loss": 0.6551851034164429, "acc": 0.6259429454803467, "length": 1.001198649406433, "mode": "train", "epoch": 35}
{"loss": 0.66060471534729, "acc": 0.6232423186302185, "length": 1.0, "mode": "test", "epoch": 35}
{"loss": 0.6546995043754578, "acc": 0.6266326904296875, "length": 1.0011534690856934, "mode": "train", "epoch": 36}
{"loss": 0.6599677205085754, "acc": 0.6246094107627869, "length": 1.0, "mode": "test", "epoch": 36}
{"loss": 0.6550070643424988, "acc": 0.626522958278656, "length": 1.0010448694229126, "mode": "train", "epoch": 37}
{"loss": 0.6594901084899902, "acc": 0.6228516101837158, "length": 1.0, "mode": "test", "epoch": 37}
{"loss": 0.6549602150917053, "acc": 0.6264636516571045, "length": 1.0008831024169922, "mode": "train", "epoch": 38}
{"loss": 0.6617088317871094, "acc": 0.6224609017372131, "length": 1.0, "mode": "test", "epoch": 38}
{"loss": 0.6549287438392639, "acc": 0.6253103017807007, "length": 1.001626968383789, "mode": "train", "epoch": 39}
{"loss": 0.6612029075622559, "acc": 0.6281249523162842, "length": 1.0, "mode": "test", "epoch": 39}
{"loss": 0.6548541188240051, "acc": 0.6249707341194153, "length": 1.0007306337356567, "mode": "train", "epoch": 40}
{"loss": 0.6603966951370239, "acc": 0.618945300579071, "length": 1.0, "mode": "test", "epoch": 40}
{"loss": 0.655652642250061, "acc": 0.6241061091423035, "length": 1.0007660388946533, "mode": "train", "epoch": 41}
{"loss": 0.6622977256774902, "acc": 0.6154298186302185, "length": 1.0, "mode": "test", "epoch": 41}
{"loss": 0.6548755764961243, "acc": 0.6263426542282104, "length": 1.0010913610458374, "mode": "train", "epoch": 42}
{"loss": 0.660144031047821, "acc": 0.6224609017372131, "length": 1.0, "mode": "test", "epoch": 42}
{"loss": 0.6551430821418762, "acc": 0.6269251704216003, "length": 1.0007350444793701, "mode": "train", "epoch": 43}
{"loss": 0.6620384454727173, "acc": 0.6205077767372131, "length": 1.0, "mode": "test", "epoch": 43}
{"loss": 0.655299186706543, "acc": 0.6260619759559631, "length": 1.0005388259887695, "mode": "train", "epoch": 44}
{"loss": 0.6600759029388428, "acc": 0.6226562857627869, "length": 1.0, "mode": "test", "epoch": 44}
{"loss": 0.6541014313697815, "acc": 0.6269264817237854, "length": 1.0004820823669434, "mode": "train", "epoch": 45}
{"loss": 0.6632761359214783, "acc": 0.6158203482627869, "length": 1.0, "mode": "test", "epoch": 45}
{"loss": 0.6553273797035217, "acc": 0.6274431347846985, "length": 1.0004243850708008, "mode": "train", "epoch": 46}
{"loss": 0.6613942980766296, "acc": 0.618359386920929, "length": 1.0, "mode": "test", "epoch": 46}
{"loss": 0.6552167534828186, "acc": 0.6269233226776123, "length": 1.0006110668182373, "mode": "train", "epoch": 47}
{"loss": 0.6596097350120544, "acc": 0.6265625357627869, "length": 1.0, "mode": "test", "epoch": 47}
{"loss": 0.6538271307945251, "acc": 0.6268108487129211, "length": 1.000349521636963, "mode": "train", "epoch": 48}
{"loss": 0.6607721447944641, "acc": 0.6261720061302185, "length": 1.0, "mode": "test", "epoch": 48}
{"loss": 0.6544986963272095, "acc": 0.627156674861908, "length": 1.0003671646118164, "mode": "train", "epoch": 49}
{"loss": 0.6613852381706238, "acc": 0.6228516697883606, "length": 1.0, "mode": "test", "epoch": 49}
{"loss": 0.6540412902832031, "acc": 0.6265243291854858, "length": 1.0005443096160889, "mode": "train", "epoch": 50}
{"loss": 0.6604337692260742, "acc": 0.623828113079071, "length": 1.0, "mode": "test", "epoch": 50}
{"loss": 0.6545629501342773, "acc": 0.6260059475898743, "length": 1.0002470016479492, "mode": "train", "epoch": 51}
{"loss": 0.6617591977119446, "acc": 0.6166015267372131, "length": 1.0, "mode": "test", "epoch": 51}
{"loss": 0.6540361046791077, "acc": 0.6268118023872375, "length": 1.0002830028533936, "mode": "train", "epoch": 52}
{"loss": 0.6617083549499512, "acc": 0.6207031607627869, "length": 1.0, "mode": "test", "epoch": 52}
{"loss": 0.6542807221412659, "acc": 0.626350998878479, "length": 1.0002460479736328, "mode": "train", "epoch": 53}
{"loss": 0.6608094573020935, "acc": 0.6197265982627869, "length": 1.0, "mode": "test", "epoch": 53}
{"loss": 0.6548212766647339, "acc": 0.6272140741348267, "length": 1.0002092123031616, "mode": "train", "epoch": 54}
{"loss": 0.6603927612304688, "acc": 0.6216797232627869, "length": 1.0, "mode": "test", "epoch": 54}
{"loss": 0.6538400650024414, "acc": 0.6257762312889099, "length": 1.0002361536026, "mode": "train", "epoch": 55}
{"loss": 0.6601006388664246, "acc": 0.6263671517372131, "length": 1.0, "mode": "test", "epoch": 55}
{"loss": 0.6548746824264526, "acc": 0.6262364387512207, "length": 1.0001790523529053, "mode": "train", "epoch": 56}
{"loss": 0.6613004803657532, "acc": 0.6267578601837158, "length": 1.0, "mode": "test", "epoch": 56}
{"loss": 0.6540755033493042, "acc": 0.6270992159843445, "length": 1.0001609325408936, "mode": "train", "epoch": 57}
{"loss": 0.6632798910140991, "acc": 0.6193358898162842, "length": 1.0, "mode": "test", "epoch": 57}
{"loss": 0.6550830006599426, "acc": 0.6268119215965271, "length": 1.0001556873321533, "mode": "train", "epoch": 58}
{"loss": 0.6610881090164185, "acc": 0.6179687976837158, "length": 1.0, "mode": "test", "epoch": 58}
{"loss": 0.6560322642326355, "acc": 0.6246827840805054, "length": 1.000128149986267, "mode": "train", "epoch": 59}
{"loss": 0.6625141501426697, "acc": 0.61328125, "length": 1.0, "mode": "test", "epoch": 59}
{"loss": 0.6547407507896423, "acc": 0.6253737807273865, "length": 1.0001291036605835, "mode": "train", "epoch": 60}
{"loss": 0.6648099422454834, "acc": 0.6148437857627869, "length": 1.0, "mode": "test", "epoch": 60}
test accuracy: 0.5716071128845215
